net/libfab/libfab.c

```diff
@@ -824,6 +1125,8 @@ static int libfab_ma_init(struct m0_net_transfer_mc *tm)
 
 	/* TODO: open passive ep, start poller thread 
 	*  net->ntm_dom->nd_xprt_private = (m0_fab__ep *)ep
+	* Whenever acive endpoint is created nd_xprt_private needs
```

Comment:
not required as we are already storing the pointer of the parent struct 
**_tm->ntm_dom->nd_xprt_private = ma->ftm_pep;_**  (on line 1142)